foxtrot-server/src/main/java/com/flipkart/foxtrot/server/FoxtrotServer.java

```diff
@@ -46,97 +46,119 @@
 import com.flipkart.foxtrot.server.providers.FlatResponseTextProvider;
 import com.flipkart.foxtrot.server.resources.*;
 import com.flipkart.foxtrot.sql.FqlEngine;
-import com.yammer.dropwizard.Service;
-import com.yammer.dropwizard.assets.AssetsBundle;
-import com.yammer.dropwizard.config.Bootstrap;
-import com.yammer.dropwizard.config.Environment;
-import com.yammer.metrics.core.HealthCheck;
-import net.sourceforge.cobertura.CoverageIgnore;
+import io.dropwizard.Application;
+import io.dropwizard.assets.AssetsBundle;
+import io.dropwizard.configuration.EnvironmentVariableSubstitutor;
+import io.dropwizard.configuration.SubstitutingSourceProvider;
+import io.dropwizard.server.DefaultServerFactory;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.util.Duration;
 import org.eclipse.jetty.servlets.CrossOriginFilter;
 
+import javax.servlet.DispatcherType;
+import javax.servlet.FilterRegistration;
 import java.util.ArrayList;
+import java.util.EnumSet;
 import java.util.List;
 import java.util.concurrent.ExecutorService;
-import java.util.concurrent.TimeUnit;
+
 
 /**
  * User: Santanu Sinha (santanu.sinha@flipkart.com)
  * Date: 15/03/14
  * Time: 9:38 PM
  */
+public class FoxtrotServer extends Application<FoxtrotServerConfiguration> {
 
-@CoverageIgnore
-public class FoxtrotServer extends Service<FoxtrotServerConfiguration> {
     @Override
     public void initialize(Bootstrap<FoxtrotServerConfiguration> bootstrap) {
-        bootstrap.setName("foxtrot");
-        bootstrap.addBundle(new AssetsBundle("/console/", "/"));
+        bootstrap.setConfigurationSourceProvider(
+                new SubstitutingSourceProvider(bootstrap.getConfigurationSourceProvider(),
+                        new EnvironmentVariableSubstitutor(false)
+                )
+        );
+        bootstrap.addBundle(new AssetsBundle("/console/", "/foxtrot/console", "index.html", "console"));
         bootstrap.addCommand(new InitializerCommand());
     }
 
     @Override
+    public String getName() {
+        return "foxtrot";
+    }
+
+    @Override
     public void run(FoxtrotServerConfiguration configuration, Environment environment) throws Exception {
-        configuration.getHttpConfiguration().setRootPath("/foxtrot/*");
```

Comment:
@r0goyal Yes. rootPath does not play well with AssetsBundle. All Resources have been changes to start with /foxtrot path instead.